File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer, Integer> countedValues = new HashMap<>();
        int[] result = new int[k];

        for (int i = 0; i < nums.length; i++){
            countedValues.put(nums[i], countedValues.getOrDefault(nums[i], 0) + 1);
        }

        List<int[]> array = new ArrayList<>();
        for (Map.Entry<Integer, Integer> entry : countedValues.entrySet()){
            array.add(new int[] {entry.getValue(), entry.getKey()});
        }

        array.sort((a, b) -> b[0] - a[0]);

        for (int i = 0; i < k; i++){
            result[i] = array.get(i)[1];
        }

        return result;
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        result = []
        newLength = len(nums) * 2

        for i in range(newLength):
            if i < len(nums):
                result.append(nums[i])
            else:
                result.append(nums[i - len(nums)])

        return result